### PR TITLE
Return MoPub image via AdMob's MediaView

### PR DESCRIPTION
**Pull request by chauduyphanvu1**

### CODE REVIEW

File: ThirdPartyAdapters/mopub/mopub/src/main/java/com/mopub/mobileads/dfp/adapters/MoPubAdapter.java [hunk 1]
```diff
@@ -130,67 +133,63 @@ public void onNativeLoad(NativeAd nativeAd) {
                         if (adData instanceof StaticNativeAd) {
                             final StaticNativeAd staticNativeAd = (StaticNativeAd) adData;
 
-                            if (options != null && options.shouldReturnUrlsForImageAssets()) {
+                            if (options != null) {
```
Comment (shankervennam) [inline ThirdPartyAdapters/mopub/mopub/src/main/java/com/mopub/mobileads/dfp/adapters/MoPubAdapter.java]: Since `NativeAdOptions` are optional, we don't expect publisher to pass them every time. Here, the images are being downloaded only if the options is not `null`. This results in an `onAdFailed` getting called when the options are not set by the publiser. Please update this to download the images even if the publiser is not setting the options.

File: README.md [hunk 2]
```diff
@@ -1,49 +0,0 @@
-Google Mobile Ads SDK - Android Mediation
```
Comment (shankervennam) [inline README.md]: Guessing this file has been deleted accidentally, Please undo this change.

**Comment by chauduyphanvu1** [comment]
Changes made. Thanks!
